pkg/resources/resource_table_test.go

```diff
@@ -40,30 +44,6 @@ func TestResourceTableCreate(t *testing.T) {
 			t.Fatal(err)
 		}
 	})
-
-}
-
-func TestResourceTableUpdate(t *testing.T) {
```

Comment:
Removing the update tests since this is just much easier and more accurate with the acceptance tests.